lib/AST/ASTPrinter.cpp

```diff
@@ -1865,22 +1881,6 @@ void PrintAST::printOneParameter(const ParamDecl *param, bool Curried,
     }
   }
```

Comment:
Similarly, both the InOutTypeRepr & the InOutType sides of this should be able to be dissolved now.  This probably allows this whole blob of code to go away (always a good sign :-)